universal-login-relayer/lib/integration/sql/services/MessageSQLRepository.ts

```diff
@@ -104,6 +104,7 @@ export class MessageSQLRepository implements IMessageRepository {
   }
 
   async markAsSuccess(messageHash: string, transactionHash: string) {
+    ensure(transactionHash.length === 66, InvalidTransaction, transactionHash);
```

Comment:
1. extract `ensureProperTransactionHash(transactionHash)`,  
2. Can we use more clever ethers function for that?